### DISCUSSION TITLE
DietPi-Software | BubbleUPnP Server

### DISCUSSION THREAD

**Discussion by privacyguy123** [MichaIng/DietPi#5361] [Software Request]
#### Formal software information
- Software title | BubbleUPnP Server
- Short description | ezpz to use UPnP program
- Official URL (if available) | https://bubblesoftapps.com/bubbleupnpserver2/
- GitHub URL (if available) | N/A
- (Official) install documentation | https://bubblesoftapps.com/bubbleupnpserver2/docs/linux_install.html

#### Are there similar/alternative software titles available with DietPi-Software?
- Gmediarender sort of - but not as many features

#### What makes your requested software better than the above solutions, if available?
- Far more feature rich UPnP app with integration with many Windows/Linux/Android apps

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.
- Sorry, I am totally inexperienced in bash coding.

#### Vote for this software implementation with the ⬆️ button at this post.

**Comment by MichaIng** [comment]
Many thanks for your request. Found an old one: #1188
However, it is very old, let's stick withouts with up-to-date links to install instructions etc.

I'll move this to discussions, so others can vote for it.
